sphinx/source/docs/user_guide/compat.rst

```diff
@@ -9,32 +9,28 @@ you to use the most appropriate tool for each task.
 BokehJS
 -------
 
-Bokeh generates JavaScript, and so Bokeh output can be combined with a
-wide variety of JavaScript libraries, such as `React`_. Listing
-such libraries is beyond the scope of this document; it's best just to
+Bokeh generates JavaScript. This makes its output fully compatible with
+a wide variety of JavaScript libraries, such as `React`_. Listing all
+such libraries is beyond the scope of this document; it's best to just
 try and see!
 
 
 Datashader
 ----------
 
-A major strength of Bokeh is that it copies your data from Python
-directly into the browser, which makes it possible for the user to
-interact with the represented data quickly and responsively, even
-without a live Python process running. However, for plotting millions
-or billions of points, this strength can turn into a liability,
-because current web browsers are very limited in how much data they
-can feasibly work with. Moreover, the visual representation of such
-large datasets is often misleading due to `overplotting`_ and related
-issues.
-
-`Datashader`_ is a separately available Python library that
-pre-renders even the largest datasets into a fixed-size raster image
-that faithfully represents the data's distribution. Datashader
-includes tools and examples showing how to build interactive Bokeh
-plots that dynamically re-render these images when zooming and panning
-in Bokeh, making it practical to work with arbitrarily large datasets
-in a web browser.
+Bokeh lets you pass data from Python directly into a browser so you
+can interact with it quickly and responsively, even without a live
```

Comment:
I personally like to avoid metaphors of live and death for software related things, where possible. How about:

> even without Python actively running in the background.

Reply:
Maybe 

> without needing any connection to a running Python process. 

There might be any number of background Python processes running that have nothing to do with Bokeh so "without Python actively running in the background" is a little too general for me

Reply:
Though you *can* have a running bokeh server. Maybe just 

> Bokeh lets you pass data from Python directly into a browser so you
 can interact with it quickly and responsively.

and leave it at that?

Reply:
Hmm. I think my original reason for that statement was to indicate that if you use Bokeh, you can do a lot that is handled entirely by the browser (via JS), without needing to round trip to Python.  The latest version doesn't convey that there doesn't need to be such round tripping, which is important when contrasting with Datashader, where the round tripping is crucial.

Reply:
I'll resolve these notes and reword as a suggestion patch.